src/components/dialog/dialog.stories.mdx

```diff
@@ -406,6 +406,7 @@ set the padding to 0. Please see the [table below](#props) for more iformation a
 Using the `focusableContainers` prop allows elements that are not DOM children of the Dialog to be reachable with the keyboard while the Dialog is open.
 This may occasionally be useful with things like Toasts where they persist on the page until dismissed by clicking a close icon.
 
+**Please note that due to a known bug with `VoiceOver` and `Safari`, the `Toast` messages will not be read out in this example.** 
```

Comment:
I figured I'd put this message in there to prevent us getting Github issues logged for stuff we already know about. It's easily removed if you think it isn't needed.

Reply:
I still say leave it out - especially because it'll need removing once the fix is in (which I've currently got a PR in for!). It's unlikely a bug gets logged in the short time between the two PR's getting merged, and if it does, well there's at least 2 of us who will know and can prevent duplicate tickets getting logged!